selfhost/typechecker.jakt

```diff
@@ -2780,7 +2780,8 @@ struct Typechecker {
                                 mut checked_default_value: CheckedExpression? = None
 
                                 if default_value is Some(expression) {
-                                    // FIXME: The typechecker can't find expressions that are outside the scope of the enum.
+                                    //FIXME: The typechecker can't find expressions that are outside the scope of the enum.
+                                    //FIXME: Get the real safety mode from somewhere, right now it assumes it is safe.
```

Comment:
Please squash this commit into the second commit, and please format the comment correctly (`// FIXME`, not `//FIXME`)